app/src/main/java/ar/com/wolox/android/example/ui/example/login/ILoginView.java

```diff
@@ -0,0 +1,8 @@
+package ar.com.wolox.android.example.ui.example.login;
+
+/**
+ * Login view
+ */
+interface ILoginView {
+    void onUsernameSaved();
```

Comment:
Add an empty line above and below this function

Reply:
✔ **Done**

Reply:
I added just a line above but not below. Then I will talk with you about that, sorry for the inconvenience :sweat_smile: